src/redux/actions/message.js

```diff
@@ -0,0 +1,20 @@
+export const HIDE_MESSAGE = 'HIDE_MESSAGE';
```

Comment:
this can't be here since this is part of the library and not the distribution source, all code inside src is code that will be distributed